src/teams/TeamsScreen.js

```diff
@@ -4,36 +4,44 @@ import Modal from 'react-native-modal';
 import _ from 'lodash';
 import React, { Component } from 'react';
 import { View } from 'react-native';
-import { Container, Content, Tab, Tabs, Button, Text, Form, Item, Input } from 'native-base';
+import {
+  Container, Content, Tab, Tabs, Button, Text, Item, Input, Card,
+} from 'native-base';
 import { TeamMember } from './TeamMember';
 import { DailyEntry } from '../history/DailyEntry';
-import { User } from '../history/User';
+import { User } from '../history/User.Model';
+import { Invitation } from './Invitation.Model';
+import { Invitations } from './Invitations';
 
 export class TeamsScreen extends Component {
   constructor(props) {
     super(props);
     this.state = {
       teamMembers: [],
-      visibleModal: false,
+      visibleCreateTeamModal: false,
+      visibleInviteUserModal: false,
+      token: '',
```

Comment:
we shouldnt keep token in state. For now we can keep it as variable in the scope of a function